app/controllers/api/v1/audios_controller.rb

```diff
@@ -148,12 +148,14 @@ def create
     def update
       if @audio && request.body
         payload = JSON.parse(request.body.read)
-        @audio.title = payload['title'] if payload['title']
+        attributes = %w[title tags peaks]
+        attributes.each do |attribute|
+          @audio.send("#{attribute}=", payload[attribute]) if payload[attribute]
```

Comment:
Just verifying that if I'm updating an existing audio object with a title change, the tags attribute does not get cleared out, right?

Reply:
Yes, the tags attribute does not clear out. I tested this both locally and staging and the tags remain unchanged when the title is changed. Thank you for the review!